acmw-db-app/pages/api/mailer.js

```diff
@@ -0,0 +1,16 @@
+// using Twilio SendGrid's v3 Node.js Library
+// https://github.com/sendgrid/sendgrid-nodejs
+const sgMail = require('@sendgrid/mail');
```

Comment:
This should be a ES6 module, i.e. `import sgMail from '@sendgrid/mail';`. There should also be an import for dotenv and dotenv.config() - see api/hello.js as an example.

Reply:
Oh yeah I did `require` bc i wasn't sure how to test it 😖. should we make `reset.js` button link to this later? thanks I'll go look at `hello.js`

Reply:
yeah I have a ticket for connecting all the password stuff since a lot of it is being made at the same time